src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,3 +1,33 @@
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <>
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      {' Posted by  '}
+
+      {post.user !== undefined
+        ? <UserInfo user={post.user} />
+        : null}
```

Comment:
Це можна скоротити 
якщо `post.user` не порожній, то надрукуй `UserInfo`
```suggestion
      {post.user && (
        <UserInfo user={post.user} />
      )}
```